lib/cli.js

```diff
@@ -214,6 +214,21 @@ exports.run = async () => { // eslint-disable-line complexity
 
 	const match = arrify(conf.match);
 	const resolveTestsFrom = cli.input.length === 0 ? projectDir : process.cwd();
+
+	const files = cli.input.map(file => path.relative(resolveTestsFrom, path.resolve(process.cwd(), file)));
+	for (const file of files) {
+		try {
+			const stats = fs.statSync(file);
+			if (!stats.isFile()) {
+				exit(`${cli.input[files.indexOf(file)]} should be a file`);
+			}
+		} catch (error) {
+			if (error.code === 'ENOENT') {
+				exit(`No sush file: ${cli.input[files.indexOf(file)]}`);
+			}
+		}
```

Comment:
You need to rethrow any other error.

Reply:
I used `throw new Error(error)` at first but I changed it to `exit` after. I am not sure which one is more appropriate in this case because this error is an unexpected one